test/e2e/test_02_server.py

```diff
@@ -77,36 +77,36 @@ def test_lausanne_thumbnail_alpha(self, manager):
         manager.expect_status_code("/thumbs/{}.jpg/full/full/0/default.jpg".format(filename), 200)
 
     def test_concurrency(self, manager):
-        """handle many concurrent requests for different URLs"""
+        """handle many concurrent requests for different URLs (this may take a while, please be patient)"""
 
         # The command-line arguments we want to pass to ab for each process.
         
         filename = "load_test.jpx"
 
         ab_processes = [
```

Comment:
I haven't tried the current settings on `develop`. The load tests should make Sipi crash because of the no thread safe XMP lib.